DSA-Campaign-Uplift-Estimation/target/DSA-Campaign-Uplift-Estimation-1/Compare/compare.html

```diff
@@ -1,45 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Why did you delete all the target files? I think this may mess up my index.yaml file in the targets folder - can you instead just remove the target files from the commit?

Reply:
I think that's outdated

Reply:
Ok, hopefully it doesn't mess anything up - if it does though it should be an easy fix so no worries.